inspire_dojson/processors.py

```diff
@@ -24,19 +24,20 @@
 
 from __future__ import absolute_import, division, print_function
 
+from urlparse import urlsplit
 from inspire_utils.helpers import force_list
 
 from .conferences import conferences
 from .data import data
 from .experiments import experiments
-from .hep import hep
-from .hepnames import hepnames
+from .hep import hep, hep2marc
+from .hepnames import hepnames, hepnames2marc
 from .institutions import institutions
 from .jobs import jobs
 from .journals import journals
 
 
-def overdo_marc_dict(record):
+def marcxml2json(record):
```

Comment:
I would expect a function called `marcxml2json` to take a string containing marcxml and return json. Here it is reading a dictionary and returning another dictionary. That's confusing.